iosHyperskillApp/iosHyperskillApp/Assets.xcassets/Track/TopicsToDiscoverNext/track-topics-to-discover-next-skipped-topic.imageset/Contents.json

```diff
@@ -0,0 +1,25 @@
+{
+  "images" : [
+    {
+      "filename" : "track-topics-to-discover-next-skipped-topic-light.pdf",
+      "idiom" : "universal"
+    },
+    {
+      "appearances" : [
+        {
+          "appearance" : "luminosity",
+          "value" : "dark"
+        }
+      ],
+      "filename" : "track-topics-to-discover-next-skipped-topic-dark.pdf",
```

Comment:
You can add only one resource for the `skipped` icon and then in code use the appropriate tint color based on the current theme. Export from Figma only one `pdf` resource with black tint color (`#000000`). This technic is preferred because we are not making apple bundle to have much more space.

We are using adaptive image resources (light and dark mode) only for complex illustrations when we cannot programmatically update color appearance based on the current theme.